### DISCUSSION TITLE
Possibility to change disease incidence & prevalence statistics?

### DISCUSSION THREAD

**Discussion by nourabdelbaki** [synthetichealth/synthea#1446] [Q&A]
Hi everyone, 

First of all, I cannot believe I just found out about this project a couple weeks ago. It is very exciting! Thank you so much for your effort!

Second, I am hoping to use the generated data in a setting other than the US, and I see room for a lot of 'personalization' when building for [other areas](https://github.com/synthetichealth/synthea/wiki/Other-Areas), but I was wondering, and perhaps I have totally missed this, if we can also specify disease incidence and prevalence stats of diseases somewhere? In hopes of more accurately depicting an area's disease burden? 

Additionally, if you can also point out to me where I should be looking (code and/or documentation) to better understand this part. 

Thanks!

**Comment by dehall** [comment]
This is something we'd really like to support, but unfortunately there's not an easy way to specify the desired incidence/prevalence that way. 

These kinds of statistics are outcomes that result from potentially complex relationships within and between diseases, so it's very difficult if not impossible to work backwards to determine what needs to be changed in the disease modules to produce those results. In some cases it's easy for a person to work out. For example in the Appendicitis module, the target prevalence by gender is baked right into the module and you can change those numbers by editing the module. https://synthetichealth.github.io/module-builder/#appendicitis 
But then on the other hand, pregnancy for example is modeled as a result of sexual activity and contraceptive statistics, so that's much more complex to try to hit a given target. You are of course free to try, you can edit the modules in the module builder, try them out locally as much as you'd like, don't worry you won't break anything for anyone else.
 
Another possible approach is to use Keep Modules: https://github.com/synthetichealth/synthea/wiki/Keep-Patients-Module
A Keep Module lets you define some characteristic of a patient that you want and only patients matching that will be exported, for example "Active Condition Diabetes" or "Attribute x is true". This wouldn't let you directly set the prevalence of a condition, but you could combine multiple runs together to produce the statistics you're interested in. For example one run of 100 patients with Diabetes and one run of 100 patients without Diabetes will give you a 50% prevalence. This would probably work best if you only had one metric of interest though. 

In terms of documentation, the wiki is generally the best place to go to understand the features Synthea offers.

**Reply by nourabdelbaki** [reply]
Thank you for your prompt response, Dylan! 

I think it will be very valuable when this is implemented in the future. Until then, thank you for your suggestions. I will play around with them; they might help me mitigate some of the concerns I have.
